index.html

```diff
@@ -10,234 +10,252 @@
-          <div class="button-links">
-            <ul>
-              <li>html</li>
-              <li>css</li>
-              <li>javascript</li>
-            </ul>
-          </div>
-          <a href="#" class="project">See Project</a>
+          <img class="img" src="image/card-4.svg" alt="" />
         </div>
       </div>
+    </div>
 
-      <div class="card">
-        <img class="img" src="image/card-4.svg" alt="">
-        <div class="content">
-          <div class="topic">Multi-Post <br /> Stories</div>
-          <div class="sub-topic">
-            <span>CANOPY</span>
-            <span class="dot"></span>
-
-            <span>Back End Dev</span>
-            <span class="dot"></span>
-
-            <span>2015</span>
-          </div>
-          <div class="description">
-            A daily selection of privately personalized reads;
-            no accounts or sign-ups required.
-          </div>
-          <div class="button-links">
-            <ul>
-              <li>html</li>
-              <li>css</li>
-              <li>javascript</li>
-            </ul>
+    <div class="myself">
+      <div class="left">
+        <div class="description">
+          <h1 class="topic">About Myself</h1>
+          Hello I'm a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don't
+          hestiate to contact me.
+        </div>
+        <div class="menuSection">
+          <a href="#" target="_blank">LET'S CONNECT</a>
+          <div class="socialMedia">
+            <img src="image/Twitter.png" alt="" />
+            <img src="image/Linkedin.png" alt="" />
+            <img src="image/Meduim.png" alt="" />
+            <img src="image/Github.png" alt="" />
+            <img src="image/angellist.png" alt="" />
           </div>
-          <a href="#" class="project">See Project</a>
         </div>
+        <div class="project">Get my resume</div>
       </div>
-    </div>
-  </div>
 
-  <div>
-    <div class="topic">About Myself</div>
-    <div class="description">
-      Hello I'm a software developer! I can help you build a product ,
-      feature or website Look through some of my work and experience!
-      If you like what you see and have a project you need coded, don't
-      hestiate to contact me.
+      <ul>
+        <li>
+          <h1 class="topic item">Languages</h1>
+          <ul class="languages">
+            <li>
+              <img src="image/js.svg" alt="" />
+              JavaScript
+            </li>
+            <li>
+              <img src="image/html5.svg" alt="" />
+              HTML
+            </li>
+            <li>
+              <img src="image/css3.svg" alt="" />
+              CSS
+            </li>
+          </ul>
+        </li>
+        <li class="topic item">Frameworks</li>
+        <li class="topic item">Skills</li>
+      </ul>
     </div>
```

Comment:
- [ ] Great job on aligning this section but I think it would be better if you give more space around the section padding/margin to match the Figma and not be very close to the card above 

- [ ] I noticed here the alignment for the about my self in not the same as Figma so kindly fix that you can use flexbox for that 💪🏻💪🏻💪🏻

- [ ] Please add the arrow beside the skills to match the Figma design and give the skills more padding to be the same with Figma 💯💯✅✅✅

- [ ]  please here don't forget to add the curve to this section by border-radius-left property that would give your portfolio  more professionalism 💯💪🏻🔥🔥

- your design
![aboutdesign](https://user-images.githubusercontent.com/58553711/179356432-0cfba0a1-a29a-441c-bd89-43bf8f83ce9a.PNG)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/179356491-151559bc-ca9d-4ad1-8546-51fa75877e29.png)